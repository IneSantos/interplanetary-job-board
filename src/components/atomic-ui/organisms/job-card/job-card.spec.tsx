import React from "react";
import { render } from '@testing-library/react';
import JobCard from "./job-card";

describe("JobCard atom tests", () => {
  test(`JobCard component render`, () => {
    const { container } = render(
      <JobCard position="This is a position" company="This is a company" location="This is a location" contract="This is a contract" skills={[]} description="This is a description"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
