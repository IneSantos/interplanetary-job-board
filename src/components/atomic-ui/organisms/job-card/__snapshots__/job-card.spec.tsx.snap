// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobCard atom tests JobCard component render 1`] = `
<div
  class="card-wrapper"
>
  <div
    class="card-header"
  >
    <h2
      class="title"
    >
      This is a position
    </h2>
    <div
      class="job-details"
    >
      <div
        class="job-detail job-detail-company"
      >
        <i
          class="fas fa-building icon"
        />
        <span>
          This is a company
        </span>
      </div>
      <div
        class="job-detail job-detail-location"
      >
        <i
          class="fas fa-map-marker-alt icon"
        />
        <span>
          This is a location
        </span>
      </div>
      <div
        class="job-detail job-detail-contract"
      >
        <i
          class="fas fa-file-signature icon"
        />
        <span>
          This is a contract
        </span>
      </div>
    </div>
  </div>
  <div
    class="card-body"
  >
    <div
      class="description"
    >
      This is a description
    </div>
    <div
      class="skill-list"
    />
    <a
      class="job-apply"
      href=""
    >
      Apply
    </a>
  </div>
</div>
`;
